File: loianeTraining/javaBasico/exercicios-aulas-14-15/Ex13.java
import java.util.Scanner;

class Ex13 {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		System.out.println("Digite um numero entre 1 e 7");
		int dayWeek = scan.nextInt();

		String day = "";
		switch (dayWeek) {
			case 1:
				day = "Domingo";
				break;
			case 2:
				day = "Segunda";
				break;
			case 3:
				day = "Terca";
				break;
			case 4:
				day = "Quarta";
				break;
			case 5:
				day = "Quinta";
				break;
			case 6:
				day = "Sexta";
				break;
			case 7:
				day = "Sabado";
				break;
			default:
				day = "uma entrada incorreta";
		}

		System.out.println("O dia da semana digitada corresponde a " + day);
		scan.close();
	}
}

File: Loiane_Training_C#/Basico/C/Ex13.cs
using System;

public class Ex13
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite, de 1 a 7, o dia da semana.");
        int DiaDaSemana = int.Parse(Console.ReadLine());

        switch (DiaDaSemana)
        {
            case 1:
                Console.WriteLine("Domingo");
                break;
            case 2:
                Console.WriteLine("Segunda");
                break;
            case 3:
                Console.WriteLine("Terca");
                break;
            case 4:
                Console.WriteLine("Quarta");
                break;
            case 5:
                Console.WriteLine("Quinta");
                break;
            case 6:
                Console.WriteLine("Sexta");
                break;
            case 7:
                Console.WriteLine("Sabado");
                break;
            default:
                Console.WriteLine("Entrada incorreta!");
                break;
        }
    }
}
